meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6/lmp-base.cfg

```diff
@@ -24,6 +24,7 @@ CONFIG_SPL_OF_CONTROL=y
 # CONFIG_SPL_USB_GADGET is not set
 # CONFIG_SPL_DM_MMC is not set
 CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x18a
```

Comment:
I decided to keep the old value for lmp-base, however we can also adjust it if needed.
TBH 0x18a is a bit redundant, I'd suggest to keep 0x100 or decrease it even more

Reply:
Might be better to align with lmp.cfg here, so using what you suggested (0x100).